packages/m2m-gateway/src/services/eserviceTemplateService.ts

```diff
@@ -529,5 +530,103 @@ export function eserviceTemplateServiceBuilder(
 
       await pollEServiceTemplate(response, headers);
     },
+    async suspendEServiceTemplateVersion(
+      templateId: EServiceTemplateId,
+      versionId: EServiceTemplateVersionId,
+      { logger, headers }: WithLogger<M2MGatewayAppContext>
+    ): Promise<m2mGatewayApi.EServiceTemplateVersion> {
+      logger.info(
+        `Suspending version ${versionId} for eservice template with id ${templateId}`
+      );
+
+      const response =
+        await clients.eserviceTemplateProcessClient.suspendTemplateVersion(
+          undefined,
+          {
+            params: { templateId, templateVersionId: versionId },
+            headers,
+          }
+        );
+
+      const polledTemplate = await pollEServiceTemplateById(
+        templateId,
+        response.metadata,
+        headers
+      );
+      const version = retrieveEServiceTemplateVersionById(
+        polledTemplate,
+        unsafeBrandId(versionId)
```

Comment:
No need for branding here, same in other methods
```suggestion
        versionId
```